trace_viewer/base/ui/mouse_mode_selector.html

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <!--
-Copyright (c) 2013 The Chromium Authors. All rights reserved.
-Use of this source code is governed by a BSD-style license that can be
-found in the LICENSE file.
+ Copyright (c) 2015 The Chromium Authors. All rights reserved.
```

Comment:
The copyright check pattern is now

      r'.*?  Copyright (\(c\) )?%(year)s The %(project)s Authors\. '
        r'All rights reserved\.\n'
      r'.*? Use of this source code is governed by a BSD-style license that '
        r'can be\n'
      r'.*? found in the LICENSE file\.(?: \*/)?\n'

but while I can keep the year unchanged, I can't seem to remove the leading whitespace without the precommit check complaining. I'll ping @natduca to see if this is intentional